CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 2.5.6 (2020-07-16)
+- Fix #358 Completing Public API
+
+# 2.5.5 (2020-07-16)
```

Comment:
Note: It is totally ok to group several changes in the same version.

Reply:
I just saw that the change with 2.5.5 wasn't listed, so I put it in.
Do you require changes here?

Reply:
no need to change, but as package.json still had 2.5.4, it was safe to put everything in 2.5.5 (the only problem we want to avoid is to publish a version which already exists of course)